hatchways/team-bluejay
# FS: Moshe-edit_chef_menu_items-#46

Ticket #46 

Description:

- Chefs can create meals with images
- Chefs can edit their meal items from their profile page
- Chefs can add multiple meal items on their profile page
- All edited images are "cache busted" so they change immediately upon being edited

Testing:

- Create new user
- Click button and fill out meal form to become a chef
- Go to your profile page
- Add a meal item without an image and verify placeholder image shows
- Add new meal items with images
- Update the meal items
